queries/useDmQuery.ts

```diff
@@ -23,6 +23,11 @@ async function getDm(args: IDmQueryArgs) {
     includeSync: true,
   });
 
+  // This will not happen because the above throws if not found
+  if (!conversation) {
+    throw new Error("Conversation not found");
+  }
```

Comment:
can remove

Reply:
+1

Reply:
actually couldn't because of the type of the fetch function once I made that change to account for not throwing. 

Let's revisit this and talk about it. There's a lot of room for improvement here